spark/interpreter/src/main/java/org/apache/zeppelin/spark/PySparkInterpreter.java

```diff
@@ -172,6 +172,10 @@ public void setInterpreterContextInPython() {
   @Override
   protected String getPythonExec() {
     SparkConf sparkConf = getSparkConf();
+    return getPythonExec(sparkConf);
+  }
+
+  String getPythonExec(SparkConf sparkConf) {
```

Comment:
Please add a access modifier (e.g. `public`, `private`)

Reply:
Only package level is needed, so I didn't specify access modifier